lib/instrumentation/ids.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const { stringify } = require('querystring')
+
+class Ids {
+  toString () {
+    return stringify(this, ',', '=')
```

Comment:
Can we change this to space-separated a la logfmt? While there may not be a convention in Node.js, I think this is generally more standard across languages.

Reply:
done. 👍